bin/GoogleAdsCleanupServices.php

```diff
@@ -233,12 +233,34 @@ public static function remove( Event $event = null ) {
 	}
 
 	/**
-	 * Remove all services.
+	 * Remove all services and resources.
```

Comment:
💅  Super minor, but being strict this should be `Remove all unused services and resources.`